release/util/sanity_check_windows.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# This script automatically download ray and run the sanity check for Windows
+# in various Python version. This script requires conda command to exist.
+
+unset RAY_ADDRESS
```

Comment:
not sure why this is required. maybe remove?

Reply:
done